webrtc/RTCPeerConnection-addTransceiver.html

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCPeerConnection addTransceiver</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  'use strict';
+
+  /*
+   *  5.1. RTCPeerConnection Interface Extensions
+   *  partial interface RTCPeerConnection {
+   *      sequence<RTCRtpSender>      getSenders();
+   *      sequence<RTCRtpReceiver>    getReceivers();
+   *      sequence<RTCRtpTransceiver> getTransceivers();
+   *      RTCRtpTransceiver           addTransceiver((MediaStreamTrack or DOMString) trackOrKind,
+   *                                                 optional RTCRtpTransceiverInit init);
+   *      ...
+   *  };
+   *
+   *  dictionary RTCRtpTransceiverInit {
+   *      RTCRtpTransceiverDirection         direction = "sendrecv";
+   *      sequence<MediaStream>              streams;
+   *      sequence<RTCRtpEncodingParameters> sendEncodings;
+   *  };
+   */
+
+  test(t => {
+    const pc = new RTCPeerConnection();
+
+    const senders = pc.getSenders();
+    assert_array_equals([], senders, 'Expect senders to be empty array');
+
+    const receivers = pc.getReceivers();
+    assert_array_equals([], receivers, 'Expect receivers to be empty array');
+
+    const transceivers = pc.getTransceivers();
+    assert_array_equals([], transceivers, 'Expect transceivers to be empty array');
+
+  }, 'Initial peer connection should have list of zero senders, receivers and transceivers');
+
+  /*
+   *  5.1.  addTransceiver
+   *        The initial value of mid is null.
+   *
+   *        3.  If the first argument is a string, let it be kind and run the following steps:
+   *            2.  Let track be null.
+   *
+   *  5.3.  RTCRtpReceiver Interface
+   *        Create an RTCRtpReceiver
+   *        3.  Initialize track.kind to kind.
+   *        5.  Initialize track.label to the result of concatenating the string "remote " with kind.
+   *        6.  Initialize track.readyState to live.
+   *        7.  Initialize track.muted to true.
+   */
+  test(t => {
+    const pc = new RTCPeerConnection();
+
```

Comment:
I have added it but I think the assertion is needed only in cases where it can introduce false positive results, e.g. testing if calling the method throws `TypeError`. We are only noticing the weird error messages because the feature is not yet implemented, but tests are meant to stay long after the implementation is completed.

I think it is more readable if we write the test as if all features have been implemented. Otherwise future readers might get confused of why there are `assert_own_property` in some tests but not others, depending on whether the feature was implemented during the test was written. 

The alternative ways to make it consistent is to either remove the `assert_own_property` when the features are implemented, or aggressively use `assert_own_property` on all tests regardless of whether the feature have been implemented - which I find neither approache to be more compelling.